tests/unit/test.base.js

```diff
@@ -14,6 +14,11 @@
     'use strict';
 
     describe('base.js', function () {
+        beforeEach(function () {
+            fixture.setBase('tests/fixtures');
+            this.result = fixture.load('outdatedBrowser.html');
```

Comment:
there is already a variable called `fixture` using `fixture.setBase('tests/fixtures');` which might be confusing